plugins/aws/resoto_plugin_aws/resource/apigateway.py

```diff
@@ -497,6 +497,7 @@ def collect(cls: Type[AwsResource], json: List[Json], builder: GraphBuilder) ->
                                     resource_instance.resource_methods[method] = gm
                         builder.add_node(resource_instance, resource)
                         builder.add_edge(api_instance, EdgeType.default, node=resource_instance)
+        return []
```

Comment:
What is the meaning? Everything that is collected via this method? only the resource type that is asked?
looks wrong - there are instances created. Also all other places that return [].